src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -29,7 +29,7 @@
 #define UCT_RC_MLX5_AM_MAX_HDR(_qp_type) \
     UCT_IB_MLX5_MAX_BB * MLX5_SEND_WQE_BB - \
     (sizeof(struct mlx5_wqe_inl_data_seg) + \
-     sizeof(struct mlx5_wqe_data_seg) + \
+     UCT_IB_MAX_IOV * sizeof(struct mlx5_wqe_data_seg) + \
```

Comment:
2. just as an example of calculating available WQE size
3. right.. so it probably worked because max_iov is set to  self->super.config.tx_min_sge rather than UCT_IB_MLX5_AM_ZCOPY_MAX_IOV / UCT_IB_MAX_IOV